__docs__/wonder-blocks-popover/popover.accessibility.mdx

```diff
@@ -39,6 +39,9 @@ the `PopoverContent` component to reference the `title` and `content`
 elements/props. By doing this, the popover will be announced by screen readers
 as a dialog with a title and content.
 
+Alternatively, you can use `aria-label` to add screen reader accessible text to the popover dialog in case there is no
```

Comment:
Tweaked it a tiny bit @jandrade, hope thats ok
Pulling this from the other thread: 
> @catandthemachines Do you have any thoughts about this description? something to add/change?